cognite/neat/_utils/validation.py

```diff
@@ -1,23 +1,63 @@
+from collections.abc import Callable, Mapping
+from typing import Literal
+
 from pydantic import ValidationError
 from pydantic_core import ErrorDetails
 
 
-def humanize_validation_error(error: ValidationError) -> list[str]:
+def as_json_path(loc: tuple[str | int, ...]) -> str:
+    """Converts a location tuple to a JSON path.
+
+    Args:
+        loc: The location tuple to convert.
+
+    Returns:
+        A JSON path string.
+    """
+    if not loc:
+        return ""
+    # +1 to convert from 0-based to 1-based indexing
```

Comment:
Are you sure that 1-based indexing is the most natural? I think we use 0-based indexing in other APIs, The time series API returns 0-based indexes in its error messages.